tools/releases/make-release-tag.sh

```diff
@@ -1,70 +0,0 @@
-#! /usr/bin/env bash
```

Comment:
I'm guessing this is no longer used anywhere, right? (KM, circleci etc.)

Reply:
Yes I grepped in both repo couldn't find any reference